POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/MainPage.xaml.cs

```diff
@@ -17,19 +18,45 @@ public MainPage()
             InitializeComponent();
         }
 
+
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             if (e.Parameter is string x)
             {
+                var lSettings = ApplicationData.Current.LocalSettings;
+                Object firstRun = lSettings.Values["IsFirstRun"];
```

Comment:
why type is Object instead of bool?  would save you the trouble of having to cast down stream

Reply:
+1 Will also help prevent attempts at casting to wrong type later